save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/testsuite/TestSuiteFilters.kt

```diff
@@ -21,16 +21,25 @@ data class TestSuiteFilters(
     fun tagsAsList() = tags.split(DATABASE_DELIMITER).distinct()
 
     /**
+     * @param additionalParams some extra parameters that should be in query
      * @return [TestSuiteFilters] as query params for request
      */
-    fun toQueryParams() = listOf("tags" to tags, "name" to name, "language" to language)
+    @Suppress("TYPE_ALIAS")
+    fun toQueryParams(additionalParams: List<Pair<String, String>>) = listOf(
+        "tags" to tags,
+        "name" to name,
+        "language" to language
+    )
         .filter { it.second.isNotBlank() }
-        .joinToString("&") { "${it.first}=${it.second}" }
-        .let {
-            if (it.isNotBlank()) {
-                "?$it"
+        .plus(additionalParams)
```

Comment:
looks like `additionalParams` (isContest) is passed always

Reply:
Except manual requests sent by user with our api

Reply:
I found only `com.saveourtool.save.backend.controllers.TestSuitesController#getFilteredTestSuites` and `isContest` is provided there

Reply:
I have several endpoints that look pretty much the same and all of them have `isContest` as optional query param. It is good for cases when user uses our api but not frontend

Reply:
didn't get, but believe you